vortex-api/src/main/java/mil/army/usace/hec/vortex/geo/ZonalStatisticsCalculator.java

```diff
@@ -202,25 +202,65 @@ public static Map<String, Integer[]> createZoneMasks(Path pathToZoneDataset, Str
         return zoneMasks;
     }
 
-    private ZonalStatistics computeZonalStatistics(String id, Integer[] mask, float[] data) {
+    private ZonalStatistics computeZonalStatistics(String id, Integer[] mask, float[] data, List<String> statistics) {
 
         double noDataValue = grid.noDataValue();
 
+        List<Double> goodData = new ArrayList<>();
+
+        // compute average
         int count = 0;
         double sum = 0;
         for (int i = 0; i < data.length; i++) {
             double value = data[i];
             if (mask[i] == 1 && Double.compare(value, noDataValue) != 0) {
+
+                //if we've gotten to this point, it's a legit value; add it to the list of good data
+                goodData.add(value);
+
                 sum += value;
                 count++;
             }
         }
 
         double average = sum / count;
 
+        // compute 5 number summary
+        // first, gotta sort the Collection in ascending order
+        Collections.sort(goodData);
```

Comment:
Check whether `statistics` contains a `String `calling for any quartile information.  If true, sort the `Collection`.  Otherwise, there's no need to sort.  You can save some compute time.